.github/workflows/CI-python.yml

```diff
@@ -37,6 +34,16 @@ jobs:
         with:
           python-version: ${{ matrix.pythonVersion }}
 
+      - bash: brew install libomp
```

Comment:
This isn't an issue for this PR at all, but I'm just curious why lightgbm doesn't do this. Any idea? I think you've contributed to lightgbm in the past (do I remember that correctly?) so maybe you know...
Might be worth fixing this at the root (=lightgbm) to avoid such manual workarounds, although I'm happy we're covering Mac builds now. Thanks for this! (and feel free to resolve, it's just a thought)

Reply:
it's in the install docs for lightgbm in macos:
https://github.com/microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#macos